File: Virucodesoup/C++/Standard/ECB (Electronic Code Book)/driver.cpp
/* ===========================
File : driver.cpp

Credit:
- Coded by (aka in cyber) Vsio Stitched

Misc:
- Written in C++ programming language

License:
- Free to use
- May include me or not in credit if included in other project
=========================== */

#include "ecb.hpp"

using namespace std;


int main() {

	header();	
	cout << "\n\n==> START PROGRAM\n\n";
	ECB ecb = ECB();
	ecb.driver();
	cout << "\n\n==> END OF PROGRAM\n\n";
	
	return 0;
}

File: Virucodesoup/Ruby/Standard/Speaking in Tongues (Google Code Jam 2012)/driver.rb
# ===========================
# File : driver.rb
#
# Credit:
# - Coded by (aka in cyber) Vsio Stitched
#
# Misc:
# - Written in Ruby programming language
#
# License:
# - Free to use
# - May include me or not in credit if included in other project
# ===========================

require_relative "gjs_SpeakingInTongues.rb"

print "\n===== Google Jam Solver =====\n"
print "\n\n== START PROGRAM ==\n\n"

gjs = GJS_SpeakingInTongues.new
gjs.driver()

print "\n\n== END PROGRAM ==\n\n"

gets